avocado/utils/astring.py

```diff
@@ -32,11 +32,6 @@
 from six import string_types, PY3
 from six.moves import zip, xrange
 
-#: String containing all fs-unfriendly chars (Windows-fat/Linux-ext3)
-FS_UNSAFE_CHARS = '<>:"/\\|?*'
-#: Translate table to replace fs-unfriendly chars
-FS_TRANSLATE = string.maketrans(FS_UNSAFE_CHARS, "_" * len(FS_UNSAFE_CHARS))
```

Comment:
Well these were meant to be available to users so they can benefit from having the set of unfriendly chars. Also it means the `maketrans` is only executed once and not per each `string_to_safe_path` execution... Would you be against me reverting it back (including your py3 fixes)?

Reply:
So we have no users (as in code that will use that).  But if you believe that it's worth just having a definition at the public level, send a PR.  About reverting a change that *fixes* things (Avocado on Python 3) in favor of hypothetical users or minor speed optimizations, yes, I am against that any day.

Reply:
Nop, I don't want to revert the patch, it's good, but I think it's useful to have `FS_UNSAFE_CHARS` in global namespace and I think it makes sense to have the `FS_TRANSLATE` produced just once (but it doesn't have to be public, therefor `_FS_TRANSLATE` is fine by me). Which would you prefer?

Reply:
Oh I see. Feel free to send the PR with the definition, although, it's the kind of thing that *I* think we have to clean up, instead of keeping accumulating.

About having the translation table created only once, it's an optimization bet.  For instance, by putting at the global module level, it *can* be created even when it won't be used.  We can certainly do some profiling and check out what is better for Avocado common usage.  Having said all that, I really think that kind of thing is not the best use our energy.

Finally, what moves me is different than what moves someone else, so feel free to contribute what you believe is important.